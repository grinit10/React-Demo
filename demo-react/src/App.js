import React, { Component } from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar'
import TypoGraphy from '@material-ui/core/Typography'
import NavBar from './navbar';


class App extends Component {
  render() {
    return (
      <div>
        <AppBar color="primary" position="static">
          <Toolbar>
            <TypoGraphy variant="h5"
              color="inherit"
            >
              My header
           </TypoGraphy>
            <TypoGraphy variant="h5"
              color="inherit"
            >
              <NavBar></NavBar>
            </TypoGraphy>
          </Toolbar>
        </AppBar>

      </div>
    );
  }
}
export default App;