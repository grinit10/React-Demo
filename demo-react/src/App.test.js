import React from 'react';
import renderer from "react-test-renderer";
import App from './App';


test('spanshot test ', () => {
  const component = renderer.create(<App></App>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})