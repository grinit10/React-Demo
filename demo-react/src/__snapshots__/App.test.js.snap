// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spanshot test  1`] = `
<div>
  <div>
    <header
      className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <h5
          className="MuiTypography-root MuiTypography-h5 MuiTypography-colorInherit"
        >
          My header
        </h5>
        <h5
          className="MuiTypography-root MuiTypography-h5 MuiTypography-colorInherit"
        >
          <nav
            className="MuiList-root MuiList-padding"
          >
            <div
              className="MuiListItem-root MuiListItem-gutters"
              disabled={false}
            >
              <div
                className="MuiListItemText-root MuiListItemText-inset"
              >
                <span
                  className="MuiTypography-root MuiTypography-colorInherit"
                >
                  Home  
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </span>
              </div>
              <div
                className="MuiListItemText-root MuiListItemText-inset"
              >
                <span
                  className="MuiTypography-root MuiTypography-colorInherit"
                >
                  Posts 
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M18 2H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zM6 4h5v8l-2.5-1.5L6 12V4z"
                    />
                  </svg>
                </span>
              </div>
              <div
                className="MuiListItemText-root MuiListItemText-inset"
              >
                <span
                  className="MuiTypography-root MuiTypography-colorInherit"
                >
                  Contact 
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 5v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2H5c-1.11 0-2 .9-2 2zm12 4c0 1.66-1.34 3-3 3s-3-1.34-3-3 1.34-3 3-3 3 1.34 3 3zm-9 8c0-2 4-3.1 6-3.1s6 1.1 6 3.1v1H6v-1z"
                    />
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                  </svg>
                </span>
              </div>
            </div>
          </nav>
        </h5>
      </div>
    </header>
  </div>
</div>
`;
